habitat-baselines/habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -147,6 +147,10 @@ def from_config(
             else:
                 agent_name = config.habitat.simulator.agents_order[0]
 
+        target_policy = config.habitat_baselines.rl.policy.get(
```

Comment:
@xavierpuigf Our long-term goal should be to avoid habitat-lab/baselines changes that are specific to the HITL tool, but we can wait until October to tackle that cleanup.